test/integration/targets/ansible-galaxy-collection/tasks/build.yml

```diff
@@ -51,3 +51,26 @@
     that:
     - '"use --force to re-create the collection artifact" in build_existing_no_force.stderr'
     - '"Created collection for ansible_test.my_collection" in build_existing_force.stdout'
+
```

Comment:
These appear to be original tests rather than backports. The tests should be added to devel before being included in a backport for stable-2.9.